app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -798,10 +798,10 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler,
     }
 
     @VisibleForTesting
-    internal fun shouldPullToRefreshBeEnabled(): Boolean {
+    internal fun shouldPullToRefreshBeEnabled(inFullScreen: Boolean): Boolean {
         return FeatureFlags.pullToRefreshEnabled &&
                 requireContext().settings().isPullToRefreshEnabledInBrowser &&
-                !(requireActivity() as HomeActivity).isImmersive
```

Comment:
Just checking - does `isImmersive` mean the same as `inFullscreen` always?

Reply:
No, I found the hard way that `isImmersive` check is unreliable in some cases.